bunq2ynab.py

```diff
@@ -1,11 +1,19 @@
 from lib.sync import Sync
 from lib.config import config
+import os
```

Comment:
There is a new file aws_lambda.py that was meant for the AWS handler function. Why not use that instead of giving bunq2ynab.py two jobs?

Reply:
Code portability. In this case we only need to maintain a single application entry point. We can split it up if you prefer, but this looks the easiest to me.